.gitignore

```diff
@@ -0,0 +1 @@
+testmd
```

Comment:
* [ ] You need to ignore the `node_modules` folder in your `.gitignore file`.  Kindly add it also.